msys2-launcher/PKGBUILD

```diff
@@ -0,0 +1,27 @@
+# Maintainer: David Macek <david.macek.0@gmail.com>
+
+_realname="msys2-launcher"
+pkgname=("${_realname}")
+pkgver=1.0
+pkgrel=1
+pkgdesc="Helper for launching MSYS2 shells"
+arch=('x86_64' 'i686')
+license=('MIT')
+groups=('base')
+backup=({mingw32,mingw64,msys2}.ini)
+conflicts=("${_realname}-git")
+replaces=("${_realname}-git")
+depends=("mintty>=1~2.2.1")
+makedepends=('mingw-w64-cross-gcc')
+source=("https://github.com/msys2/msys2-launcher/archive/v1.0.tar.gz")
```

Comment:
Use pkgver instead hardcoding version in source URL

Reply:
Thanks. Fixed.